playbooks/aws/openshift-master/config.yml

```diff
@@ -36,7 +36,7 @@
         role: openshift_master,
         openshift_node_ips: "{{ hostvars['localhost'].openshift_node_ips | default(['']) }}",
         openshift_env: "{{ oo_env }}"
-        # TODO: openshift_public_ip: set to aws instance public ip
+        openshift_public_ip: "{{ hostvars[inventory_hostname].ec2_ip_address }}"
```

Comment:
The current host's variables are also accessible without calling into the hostvars dict.

The line could just be this:
openshift_public_ip: "{{ ec2_ip_address }}"

The GCE config playbook just has this:
openshift_public_ip: "{{ gce_public_ip }}",